omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/basic/GetBalancesSpec.groovy

```diff
@@ -0,0 +1,63 @@
+package foundation.omni.test.rpc.basic
+
+import foundation.omni.BaseRegTestSpec
+import foundation.omni.CurrencyID
+import foundation.omni.rpc.BalanceEntry
+import org.bitcoinj.core.Address
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Stepwise
+
+import static foundation.omni.CurrencyID.MSC
+import static foundation.omni.CurrencyID.TMSC
+
+
+/**
+ * Fund an address and check its balances in three different ways
+ */
+class GetBalancesSpec extends BaseRegTestSpec {
```

Comment:
@dexX7 This one tests all three balance checking APIs and demonstrates how they work.